examples/workflows/cockroach_basic_cluster/main.tf

```diff
@@ -0,0 +1,87 @@
+variable "cluster_name" {
+  type     = string
+  nullable = false
+}
+
+variable "sql_user_name" {
+  type     = string
+  nullable = false
+  default  = "maxroach"
+}
+
+# Remember that even variables marked sensitive will show up
+# in the Terraform state file. Always follow best practices
+# when managing sensitive info.
+# https://developer.hashicorp.com/terraform/tutorials/configuration-language/sensitive-variables#sensitive-values-in-state
+variable "sql_user_password" {
+  type      = string
+  nullable  = false
+  sensitive = true
+}
+
+variable "cloud_provider" {
+  type     = string
+  nullable = false
+  default  = "GCP"
+}
+
+variable "cloud_provider_regions" {
+  type     = list(string)
+  nullable = false
+  default  = ["us-central1"]
+}
+
+terraform {
+  required_providers {
+    cockroach = {
+      source = "cockroachdb/cockroach"
+    }
+  }
+}
+provider "cockroach" {
+  # export COCKROACH_API_KEY with the cockroach cloud API Key
+}
+
+resource "cockroach_cluster" "example" {
+  name           = var.cluster_name
+  cloud_provider = var.cloud_provider
+  plan           = "BASIC"
+  serverless     = {}
+  regions        = [for r in var.cloud_provider_regions : { name = r }]
+}
+
+resource "cockroach_sql_user" "example" {
+  name       = var.sql_user_name
+  password   = var.sql_user_password
+  cluster_id = cockroach_cluster.example.id
+}
+
+resource "cockroach_database" "example" {
```

Comment:
I wonder if we can give them an example of initializing `movr` or something here?

Reply:
Maybe, I think it'd be a different change, though, since it would/should cut across all our cluster types, not just Basic. @marksoper would be the person to spearhead any changes along these lines, or you could submit a PR if you have good ideas on how to improve.